DuggaSys/diagram.js

```diff
@@ -716,6 +717,14 @@ document.addEventListener('keydown', function (e)
             removeLines(contextLine);
             updateSelection();
         }
+       
+    }
+    // If the active element in DOM is an "INPUT" "SELECT" "TEXTAREA"
+    if ( /INPUT|SELECT|TEXTAREA/.test(document.activeElement.nodeName.toUpperCase())) {
+        if (e.key == keybinds.ENTER.key) {
```

Comment:
take this if and put it on row 720.

Change the conditional statement to (e.key == keybinds.ENTER.key && e.ctrlKey == keybinds.ENTER.ctrl)